apps/docs/pages/guides/database/extensions/pg_stat_statements.mdx

```diff
@@ -0,0 +1,93 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'pg_stat_statements',
+  title: 'pg_stat_statements: SQL Planning and Execution Statistics',
+  description:
+    'Track planning and execution statistics of all SQL statements executed on the database.',
+}
+
+`pg_stat_statements` is a database extension that exposes a view, of the same name, to track statistics about SQL statements executed on the database. The following table shows some of the available statistics and metadata:
+
+| Column Type                           | Description                                                                                                                  |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| userid oid (references pg_authid.oid) | OID of user who executed the statement                                                                                       |
+| dbid oid (references pg_database.oid) | OID of database in which the statement was executed                                                                          |
+| toplevel bool                         | True if the query was executed as a top-level statement (always true if pg_stat_statements.track is set to top)              |
+| queryid bigint                        | Hash code to identify identical normalized queries.                                                                          |
+| query text                            | Text of a representative statement                                                                                           |
+| plans bigint                          | Number of times the statement was planned (if pg_stat_statements.track_planning is enabled, otherwise zero)                  |
+| total_plan_time double precision      | Total time spent planning the statement, in milliseconds (if pg_stat_statements.track_planning is enabled, otherwise zero)   |
+| min_plan_time double precision        | Minimum time spent planning the statement, in milliseconds (if pg_stat_statements.track_planning is enabled, otherwise zero) |
```

Comment:
@olirice I updated this to render as:

![Screenshot 2023-02-02 at 3 33 06 PM](https://user-images.githubusercontent.com/7026076/216479610-230c752a-c087-420b-b9b2-e7e28d621082.png)

Just checking to make sure if you really wanted it to render as a literal markdown table. How it looked before:

![Screenshot 2023-02-02 at 4 12 51 PM](https://user-images.githubusercontent.com/7026076/216479770-5eb36718-7651-4e01-940b-6e2d58cc8599.png)